static/styles/lib/bootstrap/scss/_dropdown.scss

```diff
@@ -1,3 +1,11 @@
+$breakpoints: (
+  "420px": (width: 291px, max-width: 80vw, margin-left: 69px),
+  "368px": (max-width: 92.4vw, margin-left: 60px),
+  "360px": (max-width: 92.4vw, margin-left: 68px),
+  "350px": (max-width: 92vw, margin-left: 78px),
```

Comment:
Can we make a responsive design here? And then remove all these breakpoints and not have to add them in the future?